src/metrics/cyclomatic.rs

```diff
@@ -203,7 +203,19 @@ impl Cyclomatic for CppCode {
 
 impl Cyclomatic for PreprocCode {}
 impl Cyclomatic for CcommentCode {}
-impl Cyclomatic for JavaCode {}
+
+impl Cyclomatic for JavaCode {
+    fn compute(node: &Node, stats: &mut Stats) {
+        use Java::*;
+
+        match node.object().kind_id().into() {
+            If | For | While | Case | Catch | TernaryExpression | AMPAMP | PIPEPIPE => {
```

Comment:
It'd be nice in your tests to cover all the possibilities (there are not so much).

Reply:
Thank you for the advice! I extended my tests to cover all the different possibilities.